heareval/tasks/pipeline.py

```diff
@@ -10,21 +10,13 @@
 from typing import Dict, List, Optional, Set, Union
 from urllib.parse import urlparse
 
+import heareval.tasks.util.audio as audio_util
 import luigi
 import pandas as pd
-from pandas import DataFrame, Series
+from heareval.tasks.util.luigi import WorkTask, download_file, new_basedir
```

Comment:
What's up with these import moves - the order was correct before https://www.python.org/dev/peps/pep-0008/#imports